tags_exec.go

```diff
@@ -0,0 +1,45 @@
+package pongo2
+
+import (
+	"bytes"
+)
+
+type tagExecNode struct {
+	position    *Token
+	bodyWrapper *NodeWrapper
+}
+
+func (node *tagExecNode) Execute(ctx *ExecutionContext, writer TemplateWriter) *Error {
+	temp := bytes.NewBuffer(make([]byte, 0, 1024)) // 1 KiB size
+
+	err := node.bodyWrapper.Execute(ctx, temp)
+	if err != nil {
+		return err
+	}
+	templateSet := ctx.template.set
+	s := temp.String()
+	currentTemplate, err2 := templateSet.FromString(s)
```

Comment:
```suggestion
	tempBytes := temp.Bytes()
	currentTemplate, err := templateSet.FromBytes(tempBytes)
```
Change is because FromString converts string to bytes. Using FromBytes will not do conversion from temp to tempString(temp.String()) and tempString to bytes again(inside FromString)

Reply:
done.